.cspell/misc-terms.txt

```diff
@@ -1 +1,7 @@
 newbranch
+egdes
+BRANDES
+KOEPF
```

Comment:
:laughing: I think these are German.

We can keep these in `.cspell`, no need to change these.